Libraries/PeriphDrivers/Include/MAX32657/rtc.h

```diff
@@ -75,6 +75,14 @@ typedef enum {
     MXC_RTC_INT_FL_READY = MXC_F_RTC_CTRL_RDY, /**< Timer ready interrupt flag */
 } mxc_rtc_int_fl_t;
 
+/**
+ * @brief     Clock settings
+ */
+typedef enum {
+    MXC_RTC_ERTCO_CLK = 4,
+    MXC_RTC_INRO_CLK = 5,
+} mxc_rtc_clock_t;
```

Comment:
```suggestion
typedef enum {
    MXC_RTC_ERTCO_CLK = 0,
    MXC_RTC_INRO_CLK = 1,
    MXC_RTC_EXT_CLK = 2,
} mxc_rtc_clock_t;
```
`wrap_get_clock_source_instance` already does a conversion.